frontend/src/metabase/admin/databases/components/DatabaseHelpSidePanel/EmbeddedEngineDocContent/EmbeddedEngineDocContent.unit.spec.tsx

```diff
@@ -0,0 +1,45 @@
+import { waitFor } from "@testing-library/react";
+
+import { render, screen } from "__support__/ui";
+
+import { EmbeddedEngineDocContent } from "./EmbeddedEngineDocContent";
+
+jest.mock(
+  "docs/databases/connections/postgresql.md",
+  () => "Postgres Markdown content",
+);
+jest.mock("docs/databases/connections/athena.md", () => {
+  throw new Error("Failed to import file");
+});
+
+describe("EmbeddedEngineDocContent", () => {
+  it("renders markdown content", async () => {
+    render(<EmbeddedEngineDocContent engineKey="postgres" />);
+    await waitFor(() => {
+      expect(screen.getByText("Postgres Markdown content")).toBeInTheDocument();
+    });
```

Comment:
This applies to all other `await waitFor` patterns in this test. I think you can use the async `findBy...` from the testing library instead.